.github/workflows/test.yml

```diff
@@ -133,4 +133,27 @@ jobs:
       - name: Run GreenFrame Analysis
         uses: marmelab/greenframe-action@1.0.0
         env:
-          GREENFRAME_SECRET_TOKEN: ${{secrets.GREENFRAME_SECRET_TOKEN}}
\ No newline at end of file
+          GREENFRAME_SECRET_TOKEN: ${{secrets.GREENFRAME_SECRET_TOKEN}}
+
+  create-react-admin:
+    runs-on: ubuntu-latest
+    name: create-react-admin
+    needs: []
+    if: success() && (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/next')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
**question (non-blocking)**: I'm just wondering why not using the latest version`actions/checkout@v4`

Reply:
I used the same we already have